Watchman.Discord.UnitTests/Users/UsersControllerTests.cs

```diff
@@ -154,5 +154,55 @@ public void AddRole_ShouldThrowException()
             queryBusMock.Verify(x => x.Execute(It.IsAny<GetDiscordServerSafeRolesQuery>()), Times.Never);
             rolesServiceMock.Verify(x => x.AddRoleToUser(It.IsAny<IEnumerable<SafeRole>>(), It.IsAny<Contexts>(), It.IsAny<List<string>>()), Times.Never);
         }
+
+        [Test]
+        public async Task RemoveRole_ShouldRemoveRole()
+        {
+            //Arrange
+            RemoveRoleCommand command = new RemoveRoleCommand();
+            command.Roles = new List<string>();
```

Comment:
prrzy tworzeniu obiektu możesz to uzupełnić, poza tym, po co ci pusty obiekt?

Reply:
i nie wywalało null exception podczas testu